_data/upcoming_hackathons.yml

```diff
@@ -22,3 +32,8 @@
   loc: "Open Targets Hackathon"
   link: https://www.opentargets.org/hackathon
   community:
+
+- date: "November 3-7, 2025"
+  loc: "BioHackathon Europe"
+  link: https://biohackathon-europe.org/
```

Comment:
They don't seem to have a 2024 or 2025 specific page - but past years did. How odd.